### PR TITLE
add eslint config to repo to integrate with text editors

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-visual-effects bundle size difference from 3.0.0 to 094ae316852c91d452591bafafec378d269c783c
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-visual-effects bundle size difference from 3.0.0 to e7cb57d63fac8ed0194f4ae6c80cd3e39585dc5c
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-visual-effects bundle size difference from 3.0.0 to 3a45f45471b7a5925089c5e1a99cca6a8a7111f4
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-visual-effects bundle size difference from 3.0.0 to 5064b853c020df3710317980382c98b316f84874
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-visual-effects bundle size difference from 3.0.0 to ef66d97e8a172b4bdddb1d179273f96fdad87da6
No significant bundle size differences found.

File: demos/src/demo.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ if (toggle && canvas) {
 		const classes = ['demo-on', 'demo-off'];
 		const status = canvas.classList.contains(classes[0]);
 
-		canvas.classList.add(classes[+status]);
-		canvas.classList.remove(classes[+!status]);
+		canvas.classList.add(classes[Number(status)]);
+		canvas.classList.remove(classes[Number(!status)]);
```
Comment (notlee) [inline demos/src/demo.js:10]: `false` casts to `0` and `true` casts to `1`. That is interesting

**Review by notlee — APPROVED** [review]

**Pull request merged**